crypto/ct/ct_lib.c

```diff
@@ -148,6 +173,22 @@ void SCT_set0_extensions(SCT *sct, unsigned char *ext, size_t ext_len)
     sct->ext_len = ext_len;
 }
 
+int SCT_set1_extensions(SCT *sct, const unsigned char *ext, size_t ext_len)
+{
+    OPENSSL_free(sct->ext);
+    if (sct->ext == NULL || ext_len == 0) {
```

Comment:
There's actually a bug there - that should be "ext == NULL", not "sct->ext == NULL".